model/database/registry.yaml

```diff
@@ -127,238 +127,185 @@ groups:
-              deprecated: "Removed, use `other_sql` instead."
-              brief: "Deprecated, Microsoft SQL Server Compact is discontinued."
-              stability: experimental
             - id: mysql
-              value: "mysql"
-              brief: "MySQL (This value has stability level RELEASE CANDIDATE)"
+              value: "mysql"  # trademark on 'mysql'
+              brief: "[MySQL](https://www.mysql.com/)"
               stability: release_candidate
             - id: neo4j
-              value: "neo4j"
-              brief: "Neo4j"
-              stability: experimental
-            - id: netezza
-              value: "netezza"
-              brief: "Netezza"
-              stability: experimental
+              value: "neo4j"  # trademark on 'neo4j'
+              brief: "[Neo4j](https://neo4j.com/)"
+              stability: development
             - id: opensearch
-              value: "opensearch"
-              brief: "OpenSearch"
-              stability: experimental
+              value: "opensearch"  # trademark on 'opensearch'
+              brief: "[OpenSearch](https://opensearch.org/)"
+              stability: development
             - id: oracle
               value: "oracle"
-              brief: "Oracle Database"
-              stability: experimental
-            - id: pervasive
-              value: "pervasive"
-              brief: "Pervasive PSQL"
-              stability: experimental
-            - id: pointbase
-              value: "pointbase"
-              brief: "PointBase"
-              stability: experimental
+              brief: "[Oracle Database](https://www.oracle.com/database/)"
+              stability: development
             - id: postgresql
-              value: "postgresql"
-              brief: "PostgreSQL (This value has stability level RELEASE CANDIDATE)"
+              value: "postgresql"  # trademark on 'postgresql'
+              brief: "[PostgreSQL](https://www.postgresql.org/)"
               stability: release_candidate
-            - id: progress
-              value: "progress"
-              brief: "Progress Database"
-              stability: experimental
             - id: redis
-              value: "redis"
-              brief: "Redis"
-              stability: experimental
-            - id: redshift
-              value: "redshift"
-              brief: "Amazon Redshift"
-              stability: experimental
-            - id: spanner
-              value: "spanner"
-              brief: "Cloud Spanner"
-              stability: experimental
+              value: "redis"  # trademark on 'redis'
+              brief: "[Redis](https://redis.io/)"
+              stability: development
+            - id: sap.hana
+              value: "sap.hana"  # trademark on 'sap hana'
+              brief: "[SAP HANA](https://www.sap.com/products/technology-platform/hana/what-is-sap-hana.html)"
+              stability: development
+            - id: sap.maxdb
```

Comment:
these get tricky...

I think my biases are showing...

if it's a brand name I'm familiar with like informix, I prefer not prefixing it with `ibm.`

but in this case I don't mind

Reply:
I'm not a big fan of having `sap.hana` and `maxdb` both being SAP databases but following different naming patterns

Reply:
I think I could live with `hana` and `maxdb` as both top-level namespaces (related to https://github.com/open-telemetry/semantic-conventions/issues/608#issuecomment-2585006195)

Reply:
resolving, I'm coming around to your https://github.com/open-telemetry/semantic-conventions/issues/608#issuecomment-2584900801